contracts/tokens/BaseToken.sol

```diff
@@ -68,8 +68,10 @@ abstract contract BaseToken is Context, ERC721Enumerable, CommunityOwnable {
 
     // External functions
 
-    function setMaxSupply(uint256 newMaxSupply) external virtual onlyCommunityOwnerOrTokenMaster {
-        if (newMaxSupply < totalSupply()) {
+    function setMaxSupply(
+        uint256 newMaxSupply
+    ) external virtual onlyCommunityOwnerOrTokenMaster {
+        if (newMaxSupply < mintedCount()) {
```

Comment:
The change looks good to me, but the formatting seems off (which is probably why this isn't passing the linter on CI).

Can you please update this accordingly?